client/src/components/PostedOn.jsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import { Typography } from "@mui/material";
+import { format } from "date-fns";
+import { useTheme } from "@emotion/react";
```

Comment:
We're currently using the `useTheme` hook from `@mui/material` instead of `emotion`, lets be sure to be consistent with that